e2e/profile/test_profile.py

```
@@ -26,74 +172,271 @@ def test_profile():
+            print("\nScreenshots saved to /tmp/test_profile_*.png")
 
             return True
 
+        except AssertionError as e:
+            print(f"\n[ASSERTION ERROR] {e}")
+            take_screenshot(page, "profile_error_assertion", "Assertion error")
+            import traceback
+
+            traceback.print_exc()
+            return False
         except Exception as e:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)